install.sh

```diff
@@ -1,20 +1,18 @@
 #!/bin/bash
-set -v
+set -euv
 
-# Docker
+. retry.sh
+
+# Docker and Kurtosis CLI
 mkdir -m 0755 -p /etc/apt/keyrings
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor --yes -o /etc/apt/keyrings/docker.gpg
+retry 7 curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor --yes -o /etc/apt/keyrings/docker.gpg
 echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null
-apt-get -o DPkg::Lock::Timeout=60 update
-apt-get -o DPkg::Lock::Timeout=60 install -y ca-certificates curl gnupg lsb-release jq docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+retry 7 apt-get update
```

Comment:
This will retry after 1, 2, 4, 8, 16, 32 and 64 seconds which increases the chance for the lock to be released.